### PR TITLE
Ballerina 0.964.0

### PR DESCRIPTION
**Pull request by rasika**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/ballerina.rb [hunk 1]
```diff
@@ -1,18 +1,21 @@
 class Ballerina < Formula
   desc "The flexible, powerful and beautiful programming language"
-  homepage "https://ballerinalang.org/"
-  url "https://ballerinalang.org/downloads/ballerina-runtime/ballerina-0.96.0.zip"
-  sha256 "7bbaecbbce0f2132854104a7975dec7e9019ed699536dedca3499d42f908cc87"
+  homepage "https://ballerinalang.org"
+  url "https://ballerinalang.org/downloads/ballerina-tools/ballerina-tools-0.964.0.zip"
+  sha256 "0ea872b63807e7e59105a353e9f7b571d8321526e3defe6a1773dc44fb6c0c7c"
 
+  revision 1
```
Comment (fxcoudert) [inline Formula/ballerina.rb]: No revision line

File: Formula/ballerina.rb [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
   bottle :unneeded
 
   depends_on :java
 
   def install
     # Remove Windows files
     rm "bin/ballerina.bat"
+    rm "bin/composer.bat"
```
Comment (fxcoudert) [inline Formula/ballerina.rb]: Replace both lines with `rm Dir["bin/*.bat"]`

File: Formula/ballerina.rb [hunk 3]
```diff
@@ -21,17 +24,26 @@ def install
       s.gsub! /\r?/, ""
     end
 
+    inreplace ["bin/composer"] do |s|
+      # Translate composer script
+      s.gsub! /^BASE_DIR=.*$/, "BASE_DIR=#{libexec}/bin"
+      s.gsub! /^PRGDIR=.*$/, "PRGDIR=#{libexec}/bin"
+      # dos to unix (bug fix for version 2.3.11)
```
Comment (fxcoudert) [inline Formula/ballerina.rb]: What this issue reported? And what is “version 2.3.11”?

File: Formula/ballerina.rb [hunk 4]
```diff
@@ -21,17 +24,26 @@ def install
...
     (testpath/"helloWorld.bal").write <<~EOS
+      import ballerina.io;
       function main (string[] args) {
-        println("Hello, World!");
+        io:println("Hello, World!");
       }
     EOS
-    output = shell_output("#{bin}/ballerina run helloWorld.bal")
-    assert_equal "Hello, World!", output.chomp
+    system libexec/"bin/ballerina", "run", testpath/"helloWorld.bal"
```
Comment (fxcoudert) [inline Formula/ballerina.rb]: Please continue to check the output produced

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/ballerina.rb (hunk 3 above)
Comment (rasika) [reply Formula/ballerina.rb]: Borrowed it from https://github.com/Linuxbrew/homebrew-core/blob/master/Formula/akka.rb.  Seems version is not related here.

**Comment by rasika** [comment]
Will send a new PR.

**Pull request closed without merging**